src/util.cpp

```diff
@@ -653,15 +653,15 @@ bool RenameOver(fs::path src, fs::path dest)
 }
 
 /**
- * Ignores exceptions thrown by Boost's create_directory if the requested directory exists.
+ * Ignores exceptions thrown by Boost's create_directories if the requested directory exists.
```

Comment:
hm, ok. I was under the impression that this was changed in https://github.com/benma/bitcoin/commit/bac5c9cf643e9333479ac667426d0b70f8f3aa7f, but I guess I misread